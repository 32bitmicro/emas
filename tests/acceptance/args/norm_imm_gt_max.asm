lw r1, 65536
