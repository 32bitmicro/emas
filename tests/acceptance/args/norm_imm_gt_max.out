acceptance/args/norm_imm_gt_max.asm:1:13: Value 65536 is not an 16-bit signed/unsigned integer
